client/src/components/TreeDropdownField/TreeDropdownField.js

```diff
@@ -405,10 +423,15 @@ class TreeDropdownField extends Component {
    */
   renderMenu(renderMenuOptions) {
     // Build root node
-    const tree = this.getVisibleTree() || {};
+    const visibleTree = this.getVisibleTree();
+    const tree = Object.assign({}, visibleTree, {
+      // we only want to show options with a title
```

Comment:
Where is this logic coming from? Should a file with no title be hidden? A record can theoretically have an empty title.